app/components/meeting/new-meeting-modal.js

```diff
@@ -0,0 +1,226 @@
+      created: now,
+      agenda,
+      number: 1,
+      shortTitle: `Goedkeuring van het verslag van de vergadering van ${moment(
+        closestMeeting.plannedstart
+      ).format('dddd DD-MM-YYYY')}.`,
+      formallyOk: CONSTANTS.ACCEPTANCE_STATUSSES.NOT_YET_OK,
+      mandatees: [],
+      pieces: [],
+      approvals: [],
+      isApproval: true,
+      treatments: A([agendaItemTreatment]),
+    });
+    return await agendaitem.save();
+  }
+
+  @task({ drop: true })
+  *createNewSession() {
+    const {
+      isDigital,
+      extraInfo,
+      selectedKindUri,
+      meetingNumber,
+      formattedMeetingIdentifier,
+    } = this;
+    const kindUriToAdd = selectedKindUri || CONFIG.MINISTERRAAD_TYPES.DEFAULT;
+    const date = this.formatter.formatDate(null);
+    const startDate = this.startDate || date;
+    const newMeeting = this.store.createRecord('meeting', {
+      isDigital,
+      extraInfo,
+      isFinal: false,
+      plannedStart: startDate,
+      created: date,
+      kind: kindUriToAdd,
+      mainMeeting: this.selectedMainMeeting,
+      number: meetingNumber,
+      numberRepresentation: formattedMeetingIdentifier,
+    });
+
+    const closestMeeting =
+      yield this.agendaService.getClosestMeetingAndAgendaId(startDate);
+
+    try {
+      yield newMeeting.save();
+      const agenda = yield this.createAgenda(newMeeting, date);
+      if (!newMeeting.isAnnex && closestMeeting) {
+        yield this.createAgendaitemToApproveMinutes(
+          agenda,
+          newMeeting,
+          closestMeeting
+        );
+      }
+      yield this.newsletterService.createNewsItemForMeeting(newMeeting);
+      // TODO: Should fix sessionNrBug
+      // await this.agendaService.assignNewSessionNumbers();
+    } catch (err) {
+      this.toaster.error();
+    } finally {
+      this.args.didSave(newMeeting);
+    }
+  }
+
+  @action
+  selectMainMeeting(mainMeeting) {
+    const kind = CONFIG.MINISTERRAAD_TYPES.TYPES.find(
+      (minsterraad) => minsterraad.uri === this.selectedKindUri
+    );
+    const postfix = (kind && kind.postfix) || '';
+    this.selectedMainMeeting = mainMeeting;
+    this.startDate = mainMeeting.plannedStart;
+    this.meetingNumber = mainMeeting.number;
+    this.formattedMeetingIdentifier = `${mainMeeting.numberRepresentation}-${postfix}`;
+    this.extraInfo = mainMeeting.extraInfo;
+  }
+
+  @action
+  selectStartDate(val) {
+    this.startDate = this.formatter.formatDate(val);
+  }
+
+  @action
+  setKind(kind) {
+    this.selectedKindUri = kind;
+    if (!this.isAnnexMeeting) {
+      this.selectedMainMeeting = null;
+      this.generateNewNumber();
+    }
+  }
+
+  @action
+  meetingNumberChangedAction(meetingNumber) {
+    this.meetingNumber = meetingNumber;
+    this.formattedMeetingIdentifier = `VR PV ${this.currentYear}/${meetingNumber}`;
+  }
+
+  @action
+  editFormattedMeetingIdentifier() {
+    this.isEditingFormattedMeetingIdentifier = true;
+  }
+
+  @action
+  saveAction() {
```

Comment:
Name is too generic. It's not clear what the action does. Action can maybe even be merged with the method from the previous comment. If a value is given as argument, it is assigned to `this.formattedMeetingIdentifier`; otherwise the formatted identifier is automatically constructed.